docs/html/js/theme-dic.js

```diff
@@ -0,0 +1,195 @@
+    params: [
+      { example: '│ │' },
+      { example: '--production' },
+      { example: 'compressed into single bundle' },
+    ],
+  },
+  'TASKS.DESC': {
+    desc: 'The style for a task/option description in a task tree.',
+    usage: '{TASKS.BRANCH: {1}} {TASKS.NAME: {2}}    {TASKS.DESC: {3}}',
+    params: [
+      { example: '├─┬' },
+      { example: 'build' },
+      { example: 'Build all the things!' },
+    ],
+  },
+  'TASKS.CHILD': {
+    desc: 'The style for a child task name in a task tree.',
+    usage: '{TASKS.BRANCH: {1}} {TASKS.CHILD: {2}}',
+    params: [
+      { example: '│   ├──' },
+      { example: 'clean' },
+    ],
+  },
+  INFO: {
+    desc: 'The style for an information log.',
+    usage: '{TIMESTAMP}{INFO: {1}}',
+    params: [
+      { example: 'This is an information log' },
+    ],
+  },
+  WARN: {
+    desc: 'The style for a warning log.',
+    usage: '{TIMESTAMP}{WARN: {1}}',
+    params: [
+      { example: 'This is a warning log' },
+    ],
+  },
+  ERROR: {
+    desc: 'The style for an error log.',
+    usage: '{TIMESTAMP}{ERROR: {1}}',
+    params: [
+      { example: 'This is an error log' },
+    ],
+  },
+  TIMESTAMP: {
+    desc: 'The style for a time stamp.',
+    usage: '{TIMESTAMP}',
+  },
+  IF: {
+    desc: 'The template for a conditional output.',
+    usage: '{IF: {1}?{2}} … \\{1\\} is neither "" nor "false"\n' +
+           '{IF: {3}?{4}} … \\{3\\} is either "" or "false"',
+    params: [
+      { example: 'true' },
+      { example: 'An error message' },
+      { example: 'false' },
+      { example: 'Another error message' },
+    ],
+  },
+  NOW: {
+    desc: 'The function which returns the current datetime in the specified format.',
+    usage: '{NOW: HH:mm:ss}',
+  },
+  gray: {
+    desc: 'The function which adds gray color to a text.',
+    params: [
+      { example: 'This is a gray message.' },
+    ],
+  },
+  magenta: {
+    desc: 'The function which adds magenta color to a text.',
+    params: [
+      { example: 'This is a magenta message.' },
+    ],
+  },
+  cyan: {
+    desc: 'The function which adds gray color to a text.',
+    params: [
+      { example: 'This is a cyan message.' },
+    ],
+  },
+  red: {
+    desc: 'The function which adds red color to a text.',
+    params: [
+      { example: 'This is a red message.' },
+    ],
+  },
+  blue: {
+    desc: 'The function which adds blue color to a text.',
+    params: [
+      { example: 'This is a blue message.' },
+    ],
+  },
+  yellow: {
+    desc: 'The function which adds yellow color to a text.',
+    params: [
+      { example: 'This is a yellow message.' },
+    ],
+  },
+  bold: {
+    desc: 'The function which adds bold style to a text.',
+    params: [
+      { example: 'This is a bold message.' },
+    ],
+  },
+  code: {
+    desc: 'The function which adds gray background color to a text.',
+    params: [
+      { example: 'This is a code message.' },
+    ],
+  },
```

Comment:
Sorry. I mistook about items from `ansi-colors`. There are more items and `code` does not exist.